test/go.mod

```diff
@@ -128,3 +129,5 @@ require (
 replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.5.0
 
 replace github.com/metal3-io/baremetal-operator => ../
+
+replace github.com/metal3-io/baremetal-operator/apis => ../apis
```

Comment:
Would love input on this. We do the same in the root `go.mod` so I think this is the correct way.
At the top of the file we require BMO `v0.3.1`. This old version did not have the annotations that I used in the test. By adding this replacement, we simply use the latest BMO API, not a specific release.

Reply:
Yes, correct but IMO we should also keep the go.mod up to date as well with latest release versions so as not to confuse the user. We should perhaps do that in all the go modules.